include/wx/generic/grideditors.h

```diff
@@ -310,8 +352,23 @@ class WXDLLIMPEXP_ADV wxGridCellChoiceEditor : public wxGridCellEditor
     wxGridCellChoiceEditor(size_t count = 0,
                            const wxString choices[] = NULL,
                            bool allowOthers = false);
+
     wxGridCellChoiceEditor(const wxArrayString& choices,
-                           bool allowOthers = false);
+                           bool allowOthers = false)
+        : wxGridCellEditor(),
+          m_allowOthers(allowOthers)
+    {
+        m_choices = wxArrayString(choices);
```

Comment:
Why don't we initialize it directly? And the copy is not needed neither, i.e. it should be just
```suggestion
          m_choices(choices),
          m_allowOthers(allowOthers)
    {
```
AFAICS.